Documentation/ceph-upgrade.md

```diff
@@ -399,15 +402,9 @@ The majority of the upgrade will be handled by the Rook operator. Begin the upgr
 Ceph image field in the cluster CRD (`spec.cephVersion.image`).
 
 ```sh
-<<<<<<< HEAD
 NEW_CEPH_IMAGE='ceph/ceph:v15.2.9-20210224'
 CLUSTER_NAME="$ROOK_CLUSTER_NAMESPACE"  # change if your cluster name is not the Rook namespace
 kubectl -n $ROOK_CLUSTER_NAMESPACE patch CephCluster $CLUSTER_NAME --type=merge -p "{\"spec\": {\"cephVersion\": {\"image\": \"$NEW_CEPH_IMAGE\"}}}"
-=======
-NEW_CEPH_IMAGE='ceph/ceph:v15.2.8-20201217'
```

Comment:
hmm, where did this come from?

Reply:
@travisn I think my PR related to doc convention brought this.